src/include/sof/lib/dai.h

```diff
@@ -362,7 +362,6 @@ static inline int dai_pm_context_restore(struct dai *dai)
 {
 	int ret = dai->drv->ops.pm_context_restore(dai);
 
-
 	return ret;
```

Comment:
same here. Please check this commit. You have multiple unerlated new lines removals. Do not add them in this commit.